tests/e2e/bootstrap_with_rancher_test.go

```diff
@@ -320,5 +333,22 @@ var _ = Describe("E2E - Bootstrapping node with Rancher", Label("bootstrapping")
 			// Check that the VM is added
 			Expect(internalClusterName).To(Equal(internalClusterToken))
 		})
+
+		By("Checking VM ssh connection", func() {
+			// TODO: Create a native Go function for this
+			ip, err := exec.Command("sed", "-zn", "/name='"+vmName+"'/s/.*ip='\\(.*\\)'.*/\\1/p", "../assets/net-default.xml").CombinedOutput()
```

Comment:
is this a leftover or I am missing something here? I see it is doing a sed here, but nothing consuming the changed file

Reply:
The file is not changed, this line is to get the IP address of the node from the `default` network, as there is no DNS. The `ip` var is then used in the ssh command to run a `uname -n`.

Reply:
oh right, I was fooled by the sed command here :)